tests/unit/endpoint.spec.ts

```diff
@@ -701,11 +702,11 @@ describe("Endpoint", () => {
             inputSchema: z.object({}).transform(() => []),
             outputSchema: z.object({}),
             handler: vi.fn<any>(),
-            resultHandler: {
-              getPositiveResponse: vi.fn(),
-              getNegativeResponse: vi.fn(),
+            resultHandler: new ResultHandler({
+              positive: vi.fn(),
+              negative: [],
```

Comment:
there is a programmatic check for this in `Endpoint::constructor()`, made in https://github.com/RobinTail/express-zod-api/pull/1431

Perhaps it should be changed

Reply:
addressed. Programmatic check remains but moved to `normalize()`. Type of the used definition changed.